sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -320,6 +683,45 @@ internal static RecognizePiiEntitiesResultCollection ConvertToRecognizePiiEntiti
             throw new NotImplementedException();
         }
 
+        internal static Legacy.PiiTask ConvertToLegacyPiiTask(RecognizePiiEntitiesAction action)
+        {
+            var parameters = new Legacy.PiiTaskParameters()
+            {
+                Domain = action.DomainFilter.GetString() ?? (Legacy.Models.PiiTaskParametersDomain?)null,
+                ModelVersion = action.ModelVersion,
+                StringIndexType = Constants.DefaultLegacyStringIndexType,
+                LoggingOptOut = action.DisableServiceLogs
+            };
+
+            if (action.CategoriesFilter.Count > 0)
+            {
+                parameters.PiiCategories = new List<Legacy.PiiEntityCategory>(action.CategoriesFilter.Count);
```

Comment:
Generated code changed, with `PiiEntityCategory` moving to the `Models` namespace.  Because the new version is in the root namespace, the compiler couldn't resolve the reference properly after I regenerated.   Worked around by renaming the type to force a unique name.